cli.go

```diff
@@ -132,6 +112,52 @@ func (h *CLIHandler) Auth(c *api.Client, m map[string]string) (*api.Secret, erro
 	}
 }
 
+func callbackHandler(c *api.Client, mount string, doneCh chan<- loginResp) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		var response string
+		var secret *api.Secret
+		var err error
```

Comment:
This err is never populated, should we just strip it?

Reply:
This is addressed in https://github.com/hashicorp/vault-plugin-auth-jwt/pull/99 if we merge it.